client/src/app/components/AnalyticsProvider.tsx

```diff
@@ -25,6 +25,7 @@ export const AnalyticsProvider: React.FC<IAnalyticsProviderProps> = ({
 export const AnalyticsContextProvider: React.FC<IAnalyticsProviderProps> = ({
   children,
 }) => {
+  /* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Biome doesn't use eslint stanza style.
Shouldn't that one be replace with `// biome-ignore lint/react-hooks-rules-of-hooks` ?

Reply:
@gildub I removed the changes this PR was doing in regards of adding comments like `/* eslint-disable react-hooks/rules-of-hooks */` As you correctly mentioned it should be replaced by the Biome style.

We will need to do the migration in 2 steps:
- First step: this PR
- Second step: enable the biome lint and format rules in our CI and while doing it add the appropriate ``/* eslint-disable react-hooks/rules-of-hooks */`` or similar rules. At the moment we don't enable our Eslint rules in CI either.